.github/workflows/test_ipex.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        torch-version: ["2.2.0", "2.3.*", "2.4.*"]
-        transformers-version: ["4.39.0", "4.44.*"]
+        transformers-version: ["4.46.*"]
+        torch-version: ["2.4.0", "2.5.0"]
```

Comment:
```suggestion
        torch-version: ["2.4.0", "2.5.*"]
```

Reply:
Unfortunately, ipex sometimes only supports big versions like 2.4.0 and 2.5.0. IPEX has conflict in little versions torch like 2.5.1